pkg/cmd/driftctl.go

```diff
@@ -49,10 +52,14 @@ func NewDriftctlCmd(build build.BuildInterface) *DriftctlCmd {
 			Long:          "Detect, track and alert on infrastructure drift.",
 			SilenceErrors: true,
 			SilenceUsage:  true,
+			PreRunE:       scanCmd.PreRunE,
+			RunE:          scanCmd.RunE,
 		},
 		build,
 	}
 
+	RegisterScanFlags(&cmd.Command, &ScanOptions{BackendOptions: &backend.Options{}})
```

Comment:
It will bind flags to the root cmd, I don't think we want to do that

![image](https://user-images.githubusercontent.com/6154987/112133737-490b5200-8bcc-11eb-890d-d443aedb4e66.png)

It should stay like this for me

```
Detect, track and alert on infrastructure drift.

Usage: driftctl <command> [flags]

COMMANDS:
  completion               Generate completion script
  scan                     Scan
  version                  Display driftctl version

FLAGS:
      --error-reporting    Enable error reporting.
                           WARNING: may leak sensitive data
  -h, --help               Display help for command
      --no-version-check   Disable the version check

LEARN MORE:
  Use "driftctl <command> --help" for more information about a command

```

WDYT @cloudskiff/engineering

Reply:
I think there's a misunderstanding of what this PR is about. The goal is to be able to run the scan from the root command, which implies to add the necessary flags to it. May be you just don't want to override the root command at all? I don't personally see this as a necessary change, most people will understand the help message and use the scan command since as it's stated in the docs. But may be it could improve a bit the user experience at no cost.